packages/caliper-core/lib/common/messages/prometheusUpdateMessage.js

```diff
@@ -0,0 +1,37 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const Message = require('./message');
+const MessageTypes = require('./../utils/constants').Messages.Types;
+
+/**
+ * Class for the "prometheusUpdate" message.
+ */
+class PrometheusUpdateMessage extends Message {
```

Comment:
I think we should call this a WorkerMetricsMessage as it isn't specific to prometheus here